databuilder/databuilder/extractor/eventbridge_extractor.py

```diff
@@ -0,0 +1,189 @@
+        """
+        It gets all the schemas and yields TableMetadata
+        :return:
+        """
+        for schema_desc in self._get_raw_extract_iter(registry_name):
+            if "Content" not in schema_desc:
+                LOGGER.info(
+                    f"skipped malformatted schema: {jsonref.dumps(schema_desc)}"
+                )
+                continue
+
+            content = jsonref.loads(schema_desc["Content"])
+
+            if content.get("openapi", "") == "3.0.0":  # NOTE: OpenAPI 3.0
+                title = content["info"]["title"]
+                columns = []
+                for i, (name, schema) in enumerate(
+                    content["components"]["schemas"].items()
+                ):
+                    columns.append(
+                        ColumnMetadata(
+                            name,
+                            schema.get("description", None),
+                            self._get_property_type(schema),
+                            i,
+                        )
+                    )
+            elif (
+                content.get("$schema", "") == "http://json-schema.org/draft-04/schema#"
+            ):  # NOTE: JSON Schema Draft 4
+                title = content["title"]
+                columns = []
+                for i, (name, schema) in enumerate(content["properties"].items()):
+                    columns.append(
+                        ColumnMetadata(
+                            name,
+                            schema.get("description", None),
+                            self._get_property_type(schema),
+                            i,
+                        )
+                    )
+            else:
+                LOGGER.info(
+                    f"skipped unsupported schema format: {jsonref.dumps(schema_desc)}"
+                )
+                continue
+
+            yield TableMetadata(
+                "eventbridge",
+                cluster_name,
+                title,
+                registry_name,
+                content.get("description", None),
+                columns,
+            )
+
+    def _get_raw_extract_iter(self, registry_name: str) -> Iterator[Dict[str, Any]]:
+        """
+        Provides iterator of results row from schemas client
+        :return:
+        """
+        schemas_descs = self._search_schemas(registry_name)
+        return iter(schemas_descs)
+
+    def _search_schemas(self, registry_name: str) -> List[Dict[str, Any]]:
+        """
+        Get all schemas descriptions.
+        """
+        paginator = self._schemas.get_paginator("list_schemas")
+        schemas_names = []
+        for result in paginator.paginate(RegistryName=registry_name):
+            for schema in result["Schemas"]:
+                schemas_names.append(schema["SchemaName"])
+
+        schemas_descs = []
+        for schema_name in schemas_names:
+            schema_versions = []
+            data = self._schemas.list_schema_versions(
```

Comment:
there should be a paginator available for this https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/schemas.html#Schemas.Paginator.ListSchemaVersions